modules/javafx.graphics/src/main/java/com/sun/javafx/tk/quantum/ScrollGestureRecognizer.java

```diff
@@ -262,7 +262,7 @@ public void notifyEndTouchEvent(long time) {
         }
     }
 
-    private void sendScrollStartedEvent(double centerX, double centerY, int touchCount) {
+    private void sendScrollStartedEvent(double centerAbsX, double centerAbsY, int touchCount) {
```

Comment:
It's probably better to use other names here, as centerAbsX/Y are already used as instance variables.

Reply:
For now, I've done the first approach, given that there is no conflict with the `centerX, centerY` variables.

Reply:
I am ok with that. With this rename, the conflict is resolved.